runtime/karura/src/lib.rs

```diff
@@ -2560,6 +2560,7 @@ impl Convert<(RuntimeCall, SignedExtra), Result<(EthereumTransactionMessage, Sig
 				access_list,
 				valid_until,
 			}) => {
+				#[cfg(not(feature = "tracing"))]
```

Comment:
can we only disable the check only if the code is being traced? so that it is harder to accidentally disable this check in production execution